portal-impl/src/portal.properties

```diff
@@ -2607,6 +2608,7 @@
         en_GB,\
```

Comment:
Based on what we learned from sorting Burmese, I think en_US and en_GB should come after en_IE since the name ie: (United Kingdom) is what we sort by. Can you double check this? I'm just raising since Brian might notice.